.gitignore

```diff
@@ -1,6 +1,6 @@
 composer.phar
 vendor/
-
+.idea*
```

Comment:
Yeah we cloned it to the SDPHP Repo sometime ago to bring it to peoples attention 

https://github.com/sdphp/gitignore